### PR TITLE
add run scripts for publish deploy start stop clean on linux 

### PR DESCRIPTION
**Pull request by goksyli**

### CODE REVIEW

File: scripts/linux/deploy.simple_kv.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        ;;
+    clean)
+        run_ "clean_files"
+        ;;
+    *)
+        echo "Bug shouldn't come here"
+        echo
+        ;;
+esac
+
```
Comment (qinzuoyan) [inline scripts/linux/deploy.simple_kv.sh]: remove useless empty lines, and the same with following

File: run.sh [hunk 2]
```diff
@@ -15,6 +15,11 @@ function usage()
     echo "   start_zk    start the local single zookeeper server"
     echo "   stop_zk     stop the local single zookeeper server"
     echo "   format      check the code format"
+    echo "   publish     publish the program"
+    echo "   deploy      deploy the program to remote machine"
```
Comment (qinzuoyan) [inline run.sh:19]: what's the difference and function of "install, publish, and deploy"?

Comment (mcfatealan) [inline run.sh:19]: To my understanding (correct me if I'm wrong), a complete remote deployment process should be:
`build and install`: common meaning, in this context it specially refers to `local env`
`publish`:  gather all necessary libs, exes, config files and scripts in one folder to be `ready` to send to remote machine, this operation generates one folder for each type of apps like meta, client and replica.
`deploy`: Send the specific folder to the corresponding target machine
`start`: Start the app on the remote machine

Comment (goksyli) [inline run.sh:19]: @mcfatealan is absolutely right. I cannot agree more

Comment (qinzuoyan) [inline run.sh:19]: Nice, I have no question.

**Pull request merged**